lib/Dialect/SPIRV/Serialization/Deserializer.cpp

```diff
@@ -1779,6 +1779,53 @@ Deserializer::processOp<spirv::ExecutionModeOp>(ArrayRef<uint32_t> words) {
   return success();
 }
 
+template <>
+LogicalResult
+Deserializer::processOp<spirv::FunctionCallOp>(ArrayRef<uint32_t> operands) {
+  if (operands.size() < 3) {
+    return emitError(unknownLoc,
+                     "OpFunctionCall must have at least 3 operands");
+  }
+
+  Type resultType = getType(operands[0]);
+  if (!resultType) {
+    return emitError(unknownLoc, "undefined result type from <id> ")
+           << operands[0];
+  }
+
+  auto resultID = operands[1];
+  auto functionID = operands[2];
+  
+  auto functionName = nameMap.lookup(functionID).str();
```

Comment:
Can we have a `getFunctionSymbol` similar to `getSpecConstantSymbol` so the same logic can be reused in `processFunction`?